src/containers/load-collection.tsx

```diff
@@ -188,6 +190,17 @@ class CollectionFetcher extends React.Component<Props, State> {
     return errors ? errors.pop() : null
   }
 
+  dataDidChange(last: Record[], next: Record[]) {
+    if (last.length != next.length) {
+      return true
+    }
+
+    return last.every((left, i) => {
```

Comment:
Should this be `.some()`? i.e. "if any item in the Record list is has a different ID, return true"

Reply:
Ack. I think some and every work the same here: they'll bail early on the first instance of true/false, but actually this block should go.

I added this evaluating over-rendering, but discovered it didn't do anything.